docs/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## What Features are Being Added?
```

Comment:
I am curious to see if the creation of the PR template in the editor will allow the population for later PR's on GH